src/qoi/include/grins/rayfire_mesh.h

```diff
@@ -132,6 +134,21 @@ namespace GRINS
     //! Map of main mesh elem_id to rayfire mesh elems
     std::map<libMesh::dof_id_type,libMesh::Elem*> _elem_id_map;
 
+    //! Map to store children of rayfire EDGE2 elems
+    /*!
+    This is necessary because it is likely that on refinement,
+    a single EDGE2 rayfire element will have 3 children
```

Comment:
"This is necessary because it is possible that upon refinement, the rayfire element (EDGE2) will have more than 2 children..."